functions.php

```diff
@@ -0,0 +1,27 @@
+<?php
+function returnBooks():array
```

Comment:
You need to write tests for this function, it doesn't have input but it does have output so you can just have one test to see if you are getting the right stuff from the db (not the content because it can change but the correct fields maybe).